src/platform/packages/private/kbn-scout-reporting/src/reporting/report/events/report.ts

```diff
@@ -51,6 +56,10 @@ export class ScoutEventsReport extends ScoutReport {
   save(destination: string) {
     this.raiseIfConcluded('nothing to save because workdir has been cleared');
 
+    // Flush & close the event log file
+    fs.fsyncSync(this.eventLogFileDescriptor);
+    fs.closeSync(this.eventLogFileDescriptor);
```

Comment:
We're running the risk of leaking the file descriptor if the script terminates unexpectedly or early (e.g. if `raiseIfConcluded` throws an exception), or if the `save` method isn't called. I wonder if we should rather move the `fs.closeSync` call to the superclass' `conclude` method which is [definitely](https://github.com/elastic/kibana/blob/main/src/platform/packages/private/kbn-scout-reporting/src/reporting/jest/reporter.ts#L251) called.